extraTs/utils/Permissions.ts

```diff
@@ -1,17 +1,17 @@
-import { permissions } from '../constant';
-export default function hasPermission( moduleName: string , role: string , permissionType: string) : boolean
-{
-    console.log(permissions);
-    console.log(moduleName);
-    const temp = permissions[moduleName];
-    if(!temp || !temp[permissionType]){
-        return false;
-    }
-    if(temp['all'].includes(role)){
-        return true;
-    }
-    if(!temp[permissionType].includes(role)){
-        return false;
-    }
-    return true;
-}
+import { permissions } from '../constant';
+export default function hasPermission(moduleName,role,permissionType)
+{
+    console.log(permissions);
+    console.log(moduleName);
+    const temp = permissions[moduleName];
+    if(!temp || !temp[permissionType]){
+        return false
+    }
+    if(temp['all'].includes(role)){
+        return true;
+    }
+    if(!temp[permissionType].includes(role)){
+        return false;
+    }
+    return true;
+}
```

Comment:
use es6, proper function naming and define types for parameter

Reply:
I am used ES6 and proper naming function as well as defined parameters.